### PR TITLE
Update `README.md`

### PR DESCRIPTION
**Pull request by hvy**
Fixes an old URL and some example script in the README.

### CODE REVIEW

**Comment by hvy** [comment]
@toshihikoyanase could you have a quick look at this PR?

File: README.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ PRs to add additional projects welcome!
 
 ### Running with Optuna's Docker images?
 You can use our docker images with the tag ending with `-dev` to run most of the examples.
-For example, you can run [PyTorch Simple](./pytorch/pytorch_simple.py) via `docker run --rm optuna/optuna:py3.7-dev python examples/pytorch/pytorch_simple.py`.
+For example, you can run [PyTorch Simple](./pytorch/pytorch_simple.py) via `docker run --rm optuna/optuna:py3.7-dev python pytorch/pytorch_simple.py`.
```
Comment (toshihikoyanase) [inline README.md:120]: How about mounting the working directory to the container? 
The `examples` directory will be removed from `optuna/optuna`, and users need to get the example script somewhere else.

```suggestion
For example, you can run [PyTorch Simple](./pytorch/pytorch_simple.py) via `docker run --rm -v $(pwd):/prj -w /prj optuna/optuna:py3.7-dev python pytorch/pytorch_simple.py`.
```

Comment (hvy) [reply README.md]: You are right, thanks!

**Comment by hvy** [comment]
PTAL. 🙇

**Review by toshihikoyanase — APPROVED** [review]
Thank you for your swift action. LGTM!

**Pull request merged**